zpages: `constant 1000000000000 overflows int` build error for 32bit architectures

**Describe the bug**
I am using `go.opencensus.io/zpages` in my [project](https://github.com/sipian/prometheus/blob/opencensus-tracing/web/web.go#L449).

When I am building my project using `GOOS=${goos} GOARCH=${arch} go build .`
for the following architectures, 
`linux-386, darwin-386, windows-386, freebsd-386, openbsd-386, netbsd-386, linux-armv5, freebsd-armv6, netbsd-armv6`

I am getting the following error
`/go/src/github.com/prometheus/prometheus/vendor/go.opencensus.io/zpages/templates.go:66:9: constant 1000000000000 overflows int`


**Expected behavior**
My project should compile successfully for the above mentioned platforms.

--- a/zpages/templates.go
+++ b/zpages/templates.go
@@ -58,20 +58,24 @@ func parseTemplate(name string) *template.Template {
 }
 
 func countFormatter(num int) string {
-	if num == 0 {
+	if num <= 0 {
 		return " "
 	}
 	var floatVal float64
 	var suffix string
-	if num >= 1e12 {
-		floatVal = float64(num) / 1e9
-		suffix = " T "
-	} else if num >= 1e9 {
-		floatVal = float64(num) / 1e9
-		suffix = " G "
-	} else if num >= 1e6 {
-		floatVal = float64(num) / 1e6
+
+	num64 := int64(num)
+
+	switch {
+	case num64 <= 1e6:
+		floatVal = float64(num64) / 1e3
 		suffix = " M "
+	case num64 <= 1e9:
+		floatVal = float64(num64) / 1e6
+		suffix = " G "
+	default:
+		floatVal = float64(num64) / 1e9
+		suffix = " T "
 	}
 
 	if floatVal != 0 {

